### ISSUE TITLE
allow manual command-line input for creating users and projects

### ISSUE THREAD

**Issue by kamfonik** [CCI-MOC/moc-openstack-tools#25]
This would be useful in a number of cases:
   * testing changes to the code that don't involve the form
   * fixing a mistake
   * simple tasks like adding an existing user to a project
   * adjusting things if there was some bug in the script 

There is some basic code in the TODO at the end of addusers.py that could be adapted for this.

**Comment by ltomatsu** [comment]
Would this implementation be practical if the manual inputs would be implemented after the automation from Google sheets?

Then the users can still be added and any manual inputs would override any mistakes in the Google Sheets.

**Comment by ltomatsu** [comment]
Hi just to keep in touch I will start looking into this issue.

I understand you have two students you're overseeing so please do not feel overwhelmed with my actions!

**Comment by ltomatsu** [comment]
I was able to create manually made `rows` using `raw_inputs()`. This was able to create projects and users for both cases of a new project and an existing project. Based on this code, I would like to tackle the cases mentioned above.

I am unsure of what the first case is asking for. How are we suppose to be making test cases using the manual input if it does not involve the form? I am using the form, more specifically the header index, as the template for my test cases.

**Comment by kamfonik** [comment]
I just mean, it could be useful when you are testing some code update in your development environment, to not have to fill out the form or the spreadsheet.

**Comment by ltomatsu** [comment]
If we are filling in the entries in the command-line, is that considered useful?

**Comment by kamfonik** [comment]
Yes.  We probably won't ever use the command line for "real" entries because we want those to be recorded in the Google Sheet.  But, the 4 bullet points above are examples of when we might want to use the CLI input.

**Comment by ltomatsu** [comment]
Great!
For fixing a mistake, we can check what each input contains per row and have the option for the user to change the input.

**Comment by ltomatsu** [comment]
I added implementations for these cases 

- testing changes to the code that don't involve the form
- fixing a mistake
- simple tasks like adding an existing user to a project
